bindings/wysiwyg-wasm/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.38.0",
   "homepage": "https://gitlab.com/andybalaam/wysiwyg-rust",
   "description": "WASM bindings for wysiwyg-rust",
-  "license": "AGPL-3.0",
+  "license": "See license in README.md file in the root of this repository",
```

Comment:
This needs to have a specific format as detailed in https://docs.npmjs.com/cli/v11/configuring-npm/package-json:
```
{
  "license": "SEE LICENSE IN <filename>"
}
```
[Example in element web](https://github.com/element-hq/element-web/blob/develop/package.json#L10)